common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -71,7 +71,7 @@ trait ABTestSwitches {
     "Test whether adding deeply read articles have negative impact on recirculation",
     owners = Seq(Owner.withName("dotcom.platform")),
     safeState = Off,
-    sellByDate = Some(LocalDate.of(2022, 10, 10)),
+    sellByDate = Some(LocalDate.of(2022, 12, 12)),
```

Comment:
Also changing the day as this [should be a weekday](https://github.com/guardian/frontend/blob/main/common/test/conf/switches/SwitchesTest.scala#L94)